### PR TITLE
Add device ATIM

### PR DESCRIPTION
**Pull request by noroka68**
#### Summary

Hello,

I am Julien HESS from the company ATIM. I have added all the ATIM products. If there is a problem send me an email to julien.hess@atim.com.

I thank you in advance.

Julien HESS

ATIM

#### Changes

- ...
- ...

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/mcci/catena4612.yaml [hunk 1]
```diff
@@ -118,7 +118,7 @@ dataSheetURL: https://cdn.shopify.com/s/files/1/0873/5104/files/971001048c__Cate
 
 # Photos
 photos:
-  main: catena4612.png
```
Comment (Jaime-Trinidad) [inline vendor/mcci/catena4612.yaml:121]: please do not modify other vendor files, I will return it to the original

**Comment by Jaime-Trinidad** [comment]
Hi, thank you for adding your devices, plea Julien sign the CLA

**Comment by noroka68** [comment]
Hello,

I have a problem, I signed the CLA with my account: noroka68, however I do not know why it added me a user: Julien HESS. I can't sing with the user Julien HESS.

Is this good for you?

Or do I have to start the whole process again with a new account Julien HESS.

Thank you very much in advance,

Julien HESS

**Comment by Jaime-Trinidad** [comment]
Already Merged on #588

**Pull request closed without merging**